apps/supernova/src/components/silences/CreateSilence.tsx

```diff
@@ -211,16 +227,17 @@ const CreateSilence = ({ alert, size, variant }) => {
           title="New Silence for"
           size="large"
           open={true}
+          // @ts-ignore
           confirmButtonLabel={success ? null : "Save"}
           onCancel={() => setDisplayNewSilence(false)}
+          // @ts-ignore
           onConfirm={success ? null : onSubmitForm}
         >
           {error && <Message text={error} variant="danger" />}
 
           {success && (
             <Message className="mb-6" variant="success">
-              A silence object with id <b>{success?.silenceID}</b> was created successfully. Please note that it may
-              take up to 5 minutes for the alert to show up as silenced.
+              {/* @ts-ignore TS(2339): Property 'silenceID' does not exist on type 'never... Remove this comment to */}
```

Comment:
Now it's missing the Message content. Was:
```
A silence object with id <b>{success?.silenceID}</b> was created successfully. Please note that it may take up to 5 minutes for the alert to show up as silenced.
```

Reply:
Sorry, already pushed